### PR TITLE
Ignore commas in quotes

### PR DESCRIPTION
**Pull request by FrameBassman**
## Proposed changes
Fix for https://github.com/codeborne/selenide/issues/697 

## Checklist
- [x] Checkstyle and unit tests pass locally with my changes by running `gradle check chrome htmlunit` command
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/test/java/com/codeborne/selenide/webdriver/ChromeDriverFactoryTest.java [hunk 1]
```diff
@@ -24,13 +24,13 @@ public void tearDown() {
   @SuppressWarnings("unchecked")
   @Test
   public void transferChromeOptionArgumentsFromSystemPropsToDriver() throws IOException {
-    System.setProperty("chromeoptions.args", "abdd,--abcd,xcvcd=123");
+    System.setProperty("chromeoptions.args", "abdd,--abcd,xcvcd=123,--user-agent=\"A,B\"");
```
Comment (vinogradoff) [inline src/test/java/com/codeborne/selenide/webdriver/ChromeDriverFactoryTest.java]: please add a test for
```
--user-agent="A,B" //done
--user-agent="A B" 
--user-agent=echo\"hello world\"
--user-agent="echo\"hello,world\""
```

?

File: src/main/java/com/codeborne/selenide/webdriver/ChromeDriverFactory.java [hunk 2]
```diff
@@ -55,15 +56,29 @@ private ChromeOptions transferChromeOptionsFromSystemProperties(ChromeOptions cu
...
-                  "are supported for the chromeoptions at the moment");
+                  "are supported for the chromeoptions at the moment." +
+                  "If you want to use argument, which contains commas (e.g. user-agent) - " +
+                  "you should wrap it in quotes.");
         }
       }
     }
     return currentChromeOptions;
   }
+
```
Comment (vinogradoff) [inline src/main/java/com/codeborne/selenide/webdriver/ChromeDriverFactory.java]: comment please, what should this method actually do.

**Comment by asolntsev** [comment]
@FrameBassman Can you resolve conflicts and make requested changes?

**Comment by FrameBassman** [comment]
@asolntsev Fixed

**Comment by rosolko** [comment]
@FrameBassman hey, your test still failed:

**Comment by FrameBassman** [comment]
I changed behavior to ignoring escaped commas. @vinogradoff could you please take a look?

File: src/test/java/com/codeborne/selenide/webdriver/ChromeDriverFactoryTest.java [hunk 3]
```diff
@@ -25,20 +25,22 @@
 
   @After
   public void tearDown() {
+    System.clearProperty("chromeoptions.args");
```
[the remark this answers is not in the document]
Comment (FrameBassman) [reply src/test/java/com/codeborne/selenide/webdriver/ChromeDriverFactoryTest.java]: Fixed

**Comment by rosolko** [comment]
@FrameBassman Looks like you delete some cases in [this](https://github.com/codeborne/selenide/pull/698/commits/4324548ac65900db67b9772751f41c568384848d) commit.
Can you please revert this сases?

**Comment by rosolko** [comment]
@FrameBassman ping!

**Comment by sidelnikovmike** [comment]
Any updates?

**Comment by BorisOsipov** [comment]
@sidelnikovmike feels free to pick up this changes and makes PR

**Comment by rosolko** [comment]
Close due to inactivity and incompatibility with latest master.
Feel free to pick-up PR and made changes.

**Comment by FrameBassman** [comment]
I think that current suggested solution is not effective due to complex string processing. I suggest to escape all important commas in arguments via backslash (\) character.

**Pull request closed without merging**